Corgibytes.Freshli.Cli.Test/Functionality/Analysis/AnalysisLocationTest.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.IO;
+using Corgibytes.Freshli.Cli.Functionality.Analysis;
+using Xunit;
+
+namespace Corgibytes.Freshli.Cli.Test.Functionality.Analysis;
+
+public class AnalysisLocationTest
+{
+    [Fact]
+    public void PathWithoutCommitId()
+    {
+        var cacheDirectory = "/path/to/cache/dir";
+        var repositoryId = Guid.NewGuid().ToString();
+        var location = new AnalysisLocation(cacheDirectory, repositoryId);
+
+        var expectedPath = Path.Combine(cacheDirectory, "repositories", repositoryId);
+        Assert.Equal(expectedPath, location.Path);
+    }
```

Comment:
This is nice! I ran into this as well but didn't write a test for it, I also wasn't sure if `Path` should return a "totally" different path when commit id is given but I think in our app it works so I'm glad we had the same conclusion.